src/lang/res_tr-TR.rc

```diff
@@ -127,7 +127,7 @@ CTXMENU MENU PRELOAD
 BEGIN
     POPUP "Dummy Popup"
     BEGIN
-    MENUITEM    "Seçimden &Yeni Pencere",           IDM_NEWWINDOW
+    MENUITEM    "Seçimden &Ctrl+Shift Enter",           IDM_NEWWINDOW
     MENUITEM    "&Aç\tEnter",                       IDM_OPEN
```

Comment:
Based on google translate, the phrase Yeni Pencere is the one to keep; not the first word.